### ISSUE TITLE
Update button styling on Listings page

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#823]
As part of our design pass coming out of Accessibility feedback, we've restyled the 'See Details' button on the [Listings](https://sites-public-tj3gg4i5eq-uc.a.run.app/listings) page. The new design should have more of a rounded corner style. Note: this style should also apply to the currently-in-development #78).
![listing_button_styling](https://user-images.githubusercontent.com/394908/145229852-8592c5af-be30-4499-9ac6-a35fef2456c4.png)

Design is viewable in [Figma](https://www.figma.com/file/wAhoXVmHRIJ2TyBKFVq2JY/Desktop-and-Mobile-Experiences?node-id=3540%3A13497) (02.1).
